internal/driver/webui_test.go

```diff
@@ -119,7 +119,15 @@ func TestWebInterface(t *testing.T) {
 		for count := 0; count < 2; count++ {
 			wg.Add(1)
 			go func() {
-				http.Get(path)
+				res, err := http.Get(path)
+				if err != nil {
+					t.Error("could not fetch", c.path, err)
+					wg.Done()
```

Comment:
Consider moving these wg.Done() calls to be a single "defer wg.Done()" at the very beginning of the function. That will make sure that it is always called, regardless of how the control flow changes in the future.